Client/src/Components/Inputs/Checkbox/index.jsx

```diff
@@ -39,6 +39,7 @@ const Checkbox = ({
 }) => {
 	const sizes = { small: "14px", medium: "16px", large: "18px" };
 	const theme = useTheme();
+	const checkBoxPosition = typeof label === "string" ? {} : { pb: "65px" };
```

Comment:
This is to address the alignment issue @gorkem-bwl brought up, only when label is not a string, it will need to be a bottom padding of 65px, also not sure if this should be move to any theme , or even make it into its own css file? as it is not worth to have a separate class just for this one css property ?

Reply:
If you need additional padding for a specific case like the create infrastructure monitor page it can be added with inline styles.

I think that's a better solution than hardcoding this in. This will affect all checkboxes that use strings as labels which I don't think we want.

Reply:
I agree, the 65px seem hardcoded, we could consider a different approach. Can you bring examples for this in this thread? I don't understand the link with Gorkem's print